CHANGES/1509.removal

```diff
@@ -0,0 +1,5 @@
+Removed the manifest schema conversion machinery. If the manifest is stored locally in the newer
+format and old clients request v2 schema1 manifest they will receive 404. v2 schema1 manifest is
+still going to be mirrored from remote source during sync if available and passed to the old clients
+on the request.
+This change also removed the ``pyjwkest`` dependency, since it is not used anymore.
```

Comment:
I am not sure if this dependency removal is necessary to announce. @ipanova, what do you think?

Reply:
I agree.